docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1220,7 +1220,77 @@ _____________________
+
+Local authentication
+^^^^^^^^^^^^^^^^^^^^
+
+image::internal-eduroam-authentication.png[scaledwidth="100%",alt="Internal eduroam authentication"]
+
+1: The device connects on the eduroam SSID.
+
+2: The access point forwards the authentication request to the wireless controller.
+
+3: The controller sends the RADIUS authentication to PacketFence on port 11812.
+
+4: PacketFence checks if it's a local REALM.
+
+5: If it's local REALM, PacketFence does a NTLM request to the Active Directory (AD) domain controller to verify the identity.
+
+5a: The AD validated the credentials.
+
+5b: The AD did not validate the credentials. PacketFence sends a RADIUS Reject 
+
+5c: After a successful NTLM authentication, PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
+
+6: If it's a not local REALM, PacketFence proxies the radius request to the eduroam servers
+
+7: The eduroam servers validate the identity
+
+8: PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
+
+Configure the Eduroam source
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open the PacketFence administration web interface and go to *Configuration -> Policies and Access Control -> Authentication Sources*.
+
+Local *Exclusive Sources* and click on `New excluse source` then `Eduroam`.
+
+The information to configure that source could be found on the eduroam platform.
+
+Create the connection profile for local authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Go to *Configuration -> Policies and Access Control -> Connection Profiles -> New Connection Profile*
+
+Create a connection profile named `Local and external eduroam authentication` Check "Automatically register devices" then create a SSID filter `eduroam`. Make sure to add the Active Directory source to match on the local users.
+
+
+Inbound authentication (TLRS to PF)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+image::inbound-eduroam-authentication.png[scaledwidth="100%",alt="Inbound eduroam authentication"]
+
+1: Eduroam sends the RADIUS authentication to a public IP address (NAT/PAT) bound to PacketFence on the management IP address (Management VIP for a cluster) on port 1812.
+
+2: PacketFence forwards the NTLM request to the Active Directory.
+
+3a: Successful user identify authentication on the AD
+
+3b: NTLM request fails because of a bad identity
+
+4: PacketFence replies to the Eduroam servers either a RADIUS Access Accept for a sucessful authentication or a RADIUS access reject for an unsuccessful authentication. PacketFence sets the REALM to eduroam for all successful authentications.
+
+First, you need to refer to the previous step `Configure the Eduroam source`.
+
+Once the source is configured, you will need to create a new connection profile.
+
+Create the connection profile for outbound authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles